mne/source_estimate.py

```diff
@@ -567,10 +565,7 @@ def resample(self, sfreq, npad='auto', window='boxcar', n_jobs=1,
             Window to use in resampling. See scipy.signal.resample.
         n_jobs : int
             Number of jobs to run in parallel.
-        verbose : bool, str, int, or None
-            If not None, override default verbose level (see
-            :func:`mne.verbose` and :ref:`Logging documentation <tut_logging>`
-            for more). Defaults to self.verbose.
+        %(verbose_meth)s
```

Comment:
` Defaults to self.verbose` will be used in (almost?) all methods of our classes, so I made an entry for it.